csrankings-s.csv

```diff
@@ -781,6 +781,7 @@ Shanshan Feng,Harbin Institute of Technology,http://faculty.hitsz.edu.cn/fengsha
 Shantanu Sharma 0001,NJIT,https://web.njit.edu/~ss797,NOSCHOLARPAGE
 Shanton Chang,University of Melbourne,http://people.eng.unimelb.edu.au/slwc,saxbu9cAAAAJ
 Shaodi You,University of Amsterdam,https://youshaodi.github.io,DRoHXVMAAAAJ
+Shaofeng Zou, University at Buffalo, http://www.acsu.buffalo.edu/~szou3/

```

Comment:
Remove spaces after commas and add Google Scholar ID. Also, please clarify how you meet the inclusion criteria (able to solely advise PhD students for a degree *in Computer Science*; usually an adjunct/courtesy/cross-appointment in CS will do). Please re-open if this is the case.